apps/meteor/app/cloud/server/functions/supportedVersionsToken/supportedVersionsToken.ts

```diff
@@ -83,7 +84,10 @@ const cacheValueInSettings = <T extends SettingValue>(
 	);
 };
 
-/** CODE */
+const releaseEndpoint =
+	process.env.OVERWRITE_INTERNAL_RELEASE_URL && process.env.OVERWRITE_INTERNAL_RELEASE_URL.trim().length > 0
```

Comment:
Can it be like this?
```suggestion
const releaseEndpoint = process.env.OVERWRITE_INTERNAL_RELEASE_URL?.trim().length > 0
```

Reply:
![image](https://github.com/RocketChat/Rocket.Chat/assets/5263975/840deb95-5a37-4e83-9444-335268348c6a)